.travis.yml

```diff
@@ -17,32 +21,30 @@ cache:
     - $HOME/local
     - $HOME/swift-0.96.1
     - $HOME/julia-0.4.2
+    - $HOME/vtk-precise64
+    - $HOME/cmake-3.2.2-Linux-x86_64
 
 # environment variables
 env:
   - PYTHONPATH=~/vtk-precise64/lib/python2.7/site-packages:~/vtk-precise64/lib LD_LIBRARY_PATH=~/vtk-precise64/lib:~/local/lib:~/local/lib/R/lib
 
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+
 before_install:
   - pip install -U pip six
   - pushd "${HOME}"
-  - curl "https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-${MONGO_VERSION}.tgz" | gunzip -c | tar x
-  - cd mongodb-*/bin && export PATH="${PWD}:${PATH}"
-  - popd
-  - mkdir /tmp/db
-  - mongod --dbpath=/tmp/db >/dev/null 2>/dev/null &
-  - mongod --version
-  - pushd "${HOME}"
-  - curl -L "http://cmake.org/files/v3.2/cmake-3.2.2-Linux-x86_64.tar.gz" | gunzip -c | tar x
+  - if [ ! -f $HOME/cmake-3.2.2-Linux-x86_64/bin/cmake ] ; then  curl -L "http://cmake.org/files/v3.2/cmake-3.2.2-Linux-x86_64.tar.gz" | gunzip -c | tar x ; fi
```

Comment:
Can you remove the quotes from this url or otherwise make the URLs consistent between curls? AFP